frontend/test/specs/post/postDetailsControllerSpec.js

```diff
@@ -613,4 +617,71 @@
             expect(returnedValue).toBeFalsy();
         });
     });
+
+    describe('isSharedEvent()', () => {
+        it('should be truthy when the post is a shared_event', () => {
+            postDetailsCtrl.post = new Post({shared_event : {}});
```

Comment:
it should really return true when its an empty object?

Reply:
yep, if the field shared_event is not undefined it means it is a shared_event